cosmoz-bottom-bar-view.js

```diff
@@ -117,12 +140,12 @@
 
 			return 'padding-bottom: 0px';
 		},
-
-		_computeFixed(fixed, desktop) {
-			if (fixed === null) {
-				return desktop;
-			}
-			return fixed;
-		},
+		/**
+		 * Determine if fixed is enabled, either through fixed property or through desktop mode.
+		 * @param {boolean} fixed Fixed property.
+		 * @param {boolean} desktop Desktop mode property.
+		 * @returns {boolean} Whether fixed is enabled or not.
+		 */
+		_computeFixed: (fixed, desktop) => fixed === null ? desktop : fixed
```

Comment:
Sorry I've been wrong about this, we should use the `_computeFixed(fixed, desktop) {` form to prepare for 2 native.
But we can move to shorthand return if you want.

Reply:
Updated the declaration to the suggested declaration form. This is interesting news because this sudden new direction is closer to JSLint approved code.